doc/gh-pages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cd ..
+SOURCES="doc examples"
+git checkout gh-pages
+rm -rf _build _sources _static
+git checkout master $SOURCES
+cd doc/
+make html
+mv -fv _build/html/* ../
+cd ..
+rm -rf doc
+touch .nojekyll
```

Comment:
actually I noticed that you also have a `.nojekyll` file in this PR ... maybe let's add this line to the makefile and we are good?